cmd/deployment-check/deployment.go

```diff
@@ -251,6 +264,30 @@ func createDeployment(ctx context.Context, deploymentConfig *v1.Deployment) chan
 	return createChan
 }
 
+// createContainerConfig creates a container resource spec and returns it.
+func createToleration(t string) corev1.Toleration {
+
+	// return blank tolerations if no value provided in yaml file
+	if len(t) == 0 {
+		tolerations := corev1.Toleration{
+			Key:      "",
+			Operator: "Exists",
+			Value:    "",
+			Effect:   "",
+		}
+		return tolerations
+	}
+
+	// Create toleration Key
+	tolerations := corev1.Toleration{
+		Key:      t,
+		Operator: "Equal",
+		Value:    "kuberhealthy",
+		Effect:   "NoSchedule",
```

Comment:
same with the constant `corev1.TaintEffectNoSchedule` here for `NoSchedule`

Reply:
oh i see now....yes ignore above comment. will do